features/protected_area.feature

```diff
@@ -0,0 +1,50 @@
+Feature: As a business owner
+  In order to update the menu and prevent others form doing it
+  I want to have a section that is protected
+
+Background:
+  Given there is a user with username "bob" and password "notbob"
+
+Scenario: Business owner should be able to access login form from login link
+  Given I visit the index page
+  And I click on the "Log In" link
+  Then I should see "Username:"
+
+Scenario: Business owner should be able to log in using his credentials
+  Given I visit the index page
+  And I click on the "Log In" link
+  And I fill in "user[username]" with "bob"
+  And I fill in "user[password]" with "notbob"
+  And I click on the "click" button
```

Comment:
And you probably want to give this an id that makes more sense - like "login". Clicking the "click" button doesn't help someone understand what is going on here.